lib/audited/audit.rb

```diff
@@ -65,7 +65,7 @@ def ancestors
     def revision
       clazz = auditable_type.constantize
       (clazz.find_by_id(auditable_id) || clazz.new).tap do |m|
-        self.class.assign_revision_attributes(m, self.class.reconstruct_attributes(ancestors).merge(version: version))
+        self.class.assign_revision_attributes(m, self.class.reconstruct_attributes(ancestors).merge(audit_version: version))
```

Comment:
do we actually need this `merge` here? looks like `reconstruct_attributes` should already set it.

Reply:
Since that change was by @orenf , I'm not familiar enough to offer any insight here.

But it looks like it's been around for a long time, so I'd be inclined to leave it for now: https://github.com/jeffdill2/audited/commit/459c3132d2d8bb5ee2c9d66e084d466a8de23b79#diff-517c4cff10c88aaf70404dbac79ec85eR68